data/projects.json

```diff
@@ -1,34 +1,34 @@
 {
     "teams": [
         {
-            "name": "CX Robots",
+            "name": "Machine Translation",
             "icon": "FaRobot",
-            "description": "We focus on using NLP models to understand and automate the communication at Allegro, e.g. automatically answer questions asked to our customer support. Our main research directions are related to pretraining and evaluating large language models, semi-supervised clustering, and human-in-the-loop NLP."
+            "description": "We are developing an in-house Machine Translation engine specifically for e-commerce purposes, aiming to provide better value compared to off-the-shelf solutions. Our focus is on accurately translating industry-specific terms and jargon, while also creating a scalable and cost-efficient solution. We employ state-of-the-art machine learning methods, involving human evaluators and automatic quality estimation models to continually enhance translation quality. Our goal is to make our platform accessible to non-Polish speakers globally and contribute to the machine translation community."
+        },
+        {
+            "name": "Language Modeling",
+            "icon": "FaSitemap",
+            "description": "We employ state-of-the-art deep learning models developing NLP algorithms for a diverse set of problems that require the understanding of the natural language spoken in the e-commerce platform. We work with Large Language Models (LLMs), by fine-tuning them, sometimes pre-training them, in order to provide the company with general purpose Foundation Models that can be tailored for specific downstream tasks. Among the applications we are working at daily basis, we find: Semantic Search, Question Answering, Conversational AI, Generative AI, Named Entity Recognition."
```

Comment:
We employ state-of-the-art deep learning models and a range of NLP algorithms to solve diverse problems that require semantic understanding of the specialized language used within a unique environment of an e-commerce platform. We utilize and develop Large Language Models (LLMs), with the goal of providing the company with general purpose Foundation Models that can be tailored for specific downstream tasks. On a daily basis, we use our models in the following applications: Semantic Search, Question Answering, Conversational AI, Generative AI, Named Entity Recognition.